webapp/src/websocket_handlers.ts

```diff
@@ -185,7 +185,7 @@ export function handleUserJoined(store: Store, ev: WebSocketMessage<UserJoinedDa
     }
 
     if (ringingEnabled(store.getState()) && userID === currentUserID) {
-        const callID = calls(store.getState())[channelID].ID || '';
+        const callID = calls(store.getState())[channelID]?.ID || '';
```

Comment:
It's unrelated, but I spotted a crash, so I'm fixing it here.

Reply:
Yikes, thanks.